pulp_rpm/tests/functional/Makefile

```diff
@@ -0,0 +1,25 @@
+CPU_COUNT=$(shell python3 -c "from multiprocessing import cpu_count; print(cpu_count())")
```

Comment:
A simpler way to do this is ```nproc --all```.  ```nproc``` is part of GNU coreutils so I think it should be present on any/every system.

Not a blocker -- this works fine

Reply:
Thanks. I will take a look into this one.

Reply:
I guess `nproc` is not available on windows neither on Mac.